admin/src/containers/HomePage/hooks.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState, useRef } from 'react';
+import axios from 'axios';
+
+const useFetch = () => {
+  const isMounted = useRef(true);
+  const [state, setState] = useState({
+    error: false,
+    isLoading: true,
+    posts: [{ link: '1' }, { link: '2' }],
```

Comment:
what are these values in the array? if they are placeholders, can we move them to an external object file?